services/ske/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## v0.11.3 (2024-04-11)
+
+- Add Middleware logic to the client.
+
 ## v0.11.2 (2024-04-09)
```

Comment:
none of the releases that are part of this changelog were made on 9/04, so this change can just be a part of that release and there is no need to increment the version